src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,160 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    public class Node<T> {
+        private T element;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T value, Node<T> next) {
```

Comment:
Is it obligatory to create the constructor with prev and next parameters? Probably only value might be enough

Reply:
in my case yes / i tried first time use only contructor with value , but i got exceptions like - "u have first element null...something" and i have to initialize in each case "head" = null

Reply:
ok, got it